pkgs/applications/editors/emacs/elisp-packages/manual-packages/gn-mode/package.nix

```
@@ -0,0 +1,18 @@
+{
+  gn,
+  melpaBuild,
+}:
+
+melpaBuild {
+  pname = "gn-mode";
+  inherit (gn) src version;
+
+  files = ''("misc/emacs/gn-mode.el")'';
+
```

This fixes the missing header line error.  Ideally, someone should upstream this fix.

```suggestion
  postPatch = ''
    substituteInPlace misc/emacs/gn-mode.el --replace-fail \
      ";;; gn-mode.el - " ";;; gn-mode.el --- "
  '';
```